src/olympia/addons/templates/addons/impala/developers.html

```diff
@@ -99,9 +99,9 @@ <h1 class="addon"{{ addon.name|locale_html }}>{{ title }}</h1>
           {{ author|user_vcard }}
         </div>
         <div class="prose">
-          {% if author.bio %}
+          {% if author.biography %}
             <h3>{{ _('Who is {0}?')|f(author.name) }}</h3>
-            <p class="bio">{{ author.bio|nl2br }}</p>
+            <p class="biography">{{ author.biography|nl2br }}</p>
```

Comment:
https://github.com/mozilla/addons-server/pull/5279/commits/a602d3515ef4dcb8eacad51cb6b8005c9e1df659 but had to switch them around in https://github.com/mozilla/addons-server/pull/5279/commits/0c1373f848d08cb0fcea136880a43b0b46f5344d